### ISSUE TITLE
Feature request: LIKE operator

### ISSUE THREAD

**Issue by vmasdani** [eveningkid/denodb#242]
Hi, I've been using denodb for a while and I find it pleasant, but there's only one feature that I think is missing and it's the LIKE operator. Are there any plans to support it?

**Comment by eveningkid** [comment]
Hey Muhammad,

Great question. I just checked and here's what you could try:

```js
await Flight.where('city', 'like', '%tangerang%').get();
```

Let me know if this works. If it does, we could create shortcuts:

```js
await Flight.like('city', 'tangerang').get();
// and possibly:
await Flight.like({ city: 'tangerang', name: 'airport' }).get();
```

Keep in touch :)

**Comment by vmasdani** [comment]
> Hey Muhammad,
> 
> Great question. I just checked and here's what you could try:
> 
> ```js
> await Flight.where('city', 'like', '%tangerang%').get();
> ```
> 
> Let me know if this works. If it does, we could create shortcuts:
> 
> ```js
> await Flight.like('city', 'tangerang').get();
> // and possibly:
> await Flight.like({ city: 'tangerang', name: 'airport' }).get();
> ```
> 
> Keep in touch :)

Hi, I've tried using that method but it failed

I think it's due to the constraint of the `Operator` type in the [query builder library](https://github.com/eveningkid/denodb/blob/f1be92f59cbeda432ca74db099d7cb0ffd5ab050/lib/query-builder.ts#L6)  as there is no `like` operator. 

![Screenshot from 2021-04-25 17-44-52](https://user-images.githubusercontent.com/37009464/115990614-662da900-a5ee-11eb-8fce-ddc983f9ee10.png)

I think the fix might just be as easy as adding a `like` operator to the `Operator` type but since this library supports MongoDB I don't know if there will be any compatibility issue by just adding `like`

**Comment by vmasdani** [comment]
> > Hey Muhammad,
> > Great question. I just checked and here's what you could try:
> > ```js
> > await Flight.where('city', 'like', '%tangerang%').get();
> > ```
> > 
> > 
> > Let me know if this works. If it does, we could create shortcuts:
> > ```js
> > await Flight.like('city', 'tangerang').get();
> > // and possibly:
> > await Flight.like({ city: 'tangerang', name: 'airport' }).get();
> > ```
> > 
> > 
> > Keep in touch :)
> 
> Hi, I've tried using that method but it failed
> 
> I think it's due to the constraint of the `Operator` type in the [query builder library](https://github.com/eveningkid/denodb/blob/f1be92f59cbeda432ca74db099d7cb0ffd5ab050/lib/query-builder.ts#L6) as there is no `like` operator.
> 
> ![Screenshot from 2021-04-25 17-44-52](https://user-images.githubusercontent.com/37009464/115990614-662da900-a5ee-11eb-8fce-ddc983f9ee10.png)
> 
> I think the fix might just be as easy as adding a `like` operator to the `Operator` type but since this library supports MongoDB I don't know if there will be any compatibility issue by just adding `like`

if just adding the `like` operator is OK, i'll do a PR right away

**Comment by eveningkid** [comment]
Sure, that would do it. I'll wait for your PR then, thank you!

**Comment by eveningkid** [comment]
This has been fixed by @vmasdani work in #246 :)

Just upgrade to `1.0.35` to get the latest update.

Have a great day
